networking/using-ptp.adoc

```diff
@@ -6,24 +6,24 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-:FeatureName: Precision Time Protocol (PTP) hardware
+:FeatureName: Precision Time Protocol (PTP) hardware with single NIC configured as boundary clock
 include::snippets/technology-preview.adoc[leveloffset=+1]
 
```

Comment:
`:_content-type: <type>`

This is needed ahead of the ID statement. Make sure to replace TYPE with assembly, procedure, concept, or reference.